int32 from_drone_id
int32 to_drone_id

float64 stamp
int32[] ego_ids
int32[] other_ids
